### ISSUE TITLE
Style: consider making file upload target smaller

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/docassemble-AssemblyLine#28]
The current Docassemble upload widget overwhelms the page. My feeling is that it would be easier to read other text on the page if we made it much smaller. Hopefully this is just CSS.

**Comment by nonprofittechy** [comment]
Michelle: having a big input is something she does use. Is the documate.org/PDF one a better model?

We can fix the problem w/ clicking outside the input bringing up the "browse" window with just CSS and not get rid of the rest of the input? Should be an upstream fix as well but we can make it and submit a patch.

Also: can we find some other examples besides just documate.org/pdf and look at them side by side? Can we get a designer's help?

**Comment by purplesky2016** [comment]
I believe this issue belongs to the Weaver and was resolved by using: 
```
fields:    
  - no label: template_upload
```

**Comment by nonprofittechy** [comment]
It was resolved on the weaver but I think we could still use the CSS fix for other interviews that have file uploads. I still think it's inappropriately large for a screen like this one:

![image](https://user-images.githubusercontent.com/7645641/124004750-53e13a00-d9a6-11eb-9ae6-7d828af58e89.png)

Multiple fields on one screen, uploading an image is an optional step that not all users will take. It dominates the other fields that are more important.

**Comment by purplesky2016** [comment]
Agree, this core issue should be fixed. The weaver solution was just a workaround.
